src/components/Card/Card.tsx

```diff
@@ -154,12 +163,23 @@ const Card = React.forwardRef<HTMLLIElement, CardProps>(function cardRender(
           <ButtonList>
             {buttons.map((button, index) => (
               <li key={index}>
-                <Button
-                  onClick={onAction.bind(null, button)}
-                  onKeyPress={onAction.bind(null, button)}
-                >
-                  {button.label}
-                </Button>
+                {isHidden && (
```

Comment:
Nice to have. Possibilité de limiter le test à l'attribut tabIndex pour éviter la duplication de code

Reply:
Done